test/index.js

```diff
@@ -204,74 +213,74 @@ describe('DDWAF lifecycle', () => {
 describe('limit tests', () => {
   it('should ignore elements too far in the objects', () => {
     const waf = new DDWAF(rules)
-    const context = waf.createContext()
 
-    const result0 = context.run({
+    const context1 = waf.createContext()
+    const result1 = context1.run({
       'server.response.status': {
         a0: '404'
       }
     }, TIMEOUT)
-    assert.strictEqual(result0.action, 'monitor')
+    assert.strictEqual(result1.action, 'monitor')
+    assert(result1.data)
 
     const item = {}
     for (let i = 0; i < 1000; ++i) {
-      item[`a${i}`] = `${i}`
+      item[`a${i}`] = `${1}`
```

Comment:
Is this change from `i` to `1` intencional? If it is, why not something like that?
```suggestion
      item[`a${i}`] = '1'
```

Reply:
ooooh good catch, that was not intentional
```suggestion
      item[`a${i}`] = `${i}`
```